.github/workflows/aws_cicd.yaml

```diff
@@ -2,6 +2,11 @@ name: aws-cicd
 on:
   workflow_dispatch:
   workflow_call:
+  push:
```

Comment:
At the moment the repo has this workflow as a required status gate for merge. I'd rather we run it every time and keep the status gate than add the smarts here.

Reply:
ok. removing the smarts then
its just creates a mess when someone else does a PR too and tf goes mental